builders/integrations/relayers/gelato.md

```diff
@@ -0,0 +1,235 @@
+In this demo, we'll ask Gelato Relay SDK to call a `HelloWorld` smart contract on our behalf. The script being built is sourced from the [quick start guide](https://docs.gelato.network/developer-products/gelato-relay-sdk/quick-start){target=_blank} on Gelato Docs. Note, there is no dependency on RPC providers - once the transaction and signature are built, we simply pass them along to the Gelato Relay API. 
+
+#### Setup {: #setup }
+First, we need to import the Gelato Relay SDK and EthersJS:
+
+```
+  import { Wallet, utils } from "ethers";
+  import { GelatoRelaySDK } from "@gelatonetwork/gelato-relay-sdk";
+```
+
+#### Define the ChainID and Target Contract {: #define-the-chainid-and-target-contract }
+Next, we'll define the chain ID and the [HelloWorld contract](https://moonscan.io/address/0x3456E168d2D7271847808463D6D383D079Bd5Eaa){target=_blank} that we want to interact with.
+
+```
+  const chainId = 1284;
+  // Hello World smart contract on Moonbeam
+  const target = "0x3456E168d2D7271847808463D6D383D079Bd5Eaa";
+```
+
+#### Create a Test Account {: #create-a-test-account }
+In this step, we'll create a new test account that will submit the gasless transaction. This account is insecure and should not be used in production. We've defined a `test_token` with a default value for demonstration purposes. 
+
+```
+  const test_token = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
+  // Create Mock wallet
+  const wallet = Wallet.createRandom();
+  const sponsor = await wallet.getAddress();
+  console.log(`Mock PK: ${await wallet._signingKey().privateKey}`);
+  console.log(`Mock wallet address: ${sponsor}`);
+```
+
+
+#### Add Request Data {: #add-request-data }
+
+In this step we have to provide the ABI-encoded call data for the function we want to interact with. You can generate this by taking the following steps:
+
+1. Navigate to the **Write Contract** heading of the [Hello World Contract on Moonscan](https://moonscan.io/address/0x3456E168d2D7271847808463D6D383D079Bd5Eaa#writeContract){target=_blank} 
+2. Press  **Connect to Web3**. After you accept the terms and conditions, you can connect your wallet
+3. Head to the `sayHiVanilla` function and provide the following default value for the `_feeToken` parameter: `0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE`
+4. Press **Write**
+5. Without confirming the transaction in MetaMask, click on the **Hex** tab
+6. Press **Copy Raw Transaction Data**
+
+The resulting ABI-encodeded call data should look like `0x4b327067000000000000000000000000eeeeeeeeeeeeeeeeeeeeeeeeaeeeeeeeeeeeeeeeee`
+
+![Gelato Relay SDK](/images/builders/integrations/relayers/gelato/gelato-5.png)
+
+There are some additional parameters defined here, such as paymentType, maxFee, and gas limit. There are a variety of possible [payment types](https://docs.gelato.network/developer-products/gelato-relay-sdk/payment-types){target=_blank} you can choose from. For simplicity, replay protection has not been considered in this example. 
```

Comment:
This context switch is kind of confusing - maybe explain what you're going to do with the abi-encoded data now that you've retrieved it before jumping into additional paremeters - and where are the additional parameters defined?

Reply:
I see the parameters are defined in the following example